### ISSUE TITLE
Add support for add-ons with images on other docker registries than Dockerhub

### ISSUE THREAD

**Issue by cschroeter** [home-assistant/hassio#564]
Due to a regex check in `hassio/addons/validate.py` you can not install add-ons with images on a different registry than Dockerhub for example `registry.gitlab.com/<user>/<project>`

```python
 vol.Optional(ATTR_IMAGE): vol.Match(r"^[\w{}]+/[\-\w{}]+$"),
```

I wonder whats the reason for this behaviour?

**Comment by pvizeli** [comment]
No reason, feel free to provide a better regex :)

**Comment by sholzmayer** [comment]
i guess you can close this issue due to the merged PR #608
